src/routes/_authenticated.tsx

```
@@ -0,0 +1,55 @@
+
+  useEffect(() => {
+    const { data: { subscription } } = supabase.auth.onAuthStateChange((_event, session) => {
+      if (!session) {
+        navigate({ to: '/' })
+      }
+    })
+
+    return () => subscription.unsubscribe()
+  }, [navigate])
+
+  const handleLogout = async () => {
```

ESLint

[error] 'handleLogout' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)